www/js/app.js

```diff
@@ -112,59 +114,70 @@ angular.module('starter', ['ionic','ngMessages'])
     }
 })
 
-.controller('opencallCtrl', function ($scope, $state,$http,myCache) {
-
-        var cache = myCache.get('UserId');
-
-        if (cache) { // If there’s something in the cache, use it!
-            $scope.UserId = cache
-          }
-          else {
-            $state.go('login');
-            return;
-          }
-    $scope.calllists='';
-    $scope.calllists = [
-
-      { isDivider: true, divider: "Department A" },
-      { name: 'Welder Machine', id: 1, details: 'Opened at 10/04/2016 15:30',downtime:'false' },
-      { name: 'Press Machine', id: 2, details: 'Opened at 10/04/2016 18:30',downtime:'true' },
-        { name: 'Labeler', id: 3, details: 'Opened at 11/04/2016 15:30' ,downtime:'false'},
-       { isDivider: true, divider: "Department B" },
-
-      { name: 'Bag Machine', id: 4, details: 'Opened at 12/04/2016 15:30',downtime:'true' },
-      { name: 'Momuta', id: 5, details: 'Opened at 13/04/2016 15:30',downtime:'false'},
-
-    ];
+.controller('opencallCtrl', function ($scope, $state,$http) {
+    //variable initialization
+     var indexedTeams=[];
+     $scope.calllists='';
+     $scope.UserId=window.localStorage.getItem("UserId"); //get the login userid
+     if($scope.UserId ==undefined || $scope.UserId =="" || $scope.UserId == null)
+     {
+       $state.go('login');
+     }
 
     $scope.AddCall = function (value) {
         $state.go('app.newcall');
     }
 
+    $scope.bindcalls=function()
+    {
+
+         $http.get('http://192.168.0.137/LH_Mobile_Backend/faults/'+$scope.UserId).success(function (data) {
```

Comment:
The REST web site base address _**(http://192.168.0.137/LH_Mobile_Backend)**_ needs to be set as a general constant string like the following example:
``` javascript
const BASE_URL_VALUE = "http://192.168.0.137/LH_Mobile_Backend"
```
then anywhere in the code where we need to make a REST call we use the following syntax:
``` javascript
$http.get(BASE_URL_VALUE + '/faults/'+$scope.UserId).success(function (data) { ...
```